src/smartHandler.ts

```diff
@@ -219,7 +239,6 @@ export class SMARTHandler implements Authorization {
             }
         }
 
-        // Create a SearchFilter to limit access to only resources that are referring to the requesting user and/or context
```

Comment:
Is there a reason you removed this comment? I think it is still accurate.

Reply:
I was removing my own comments to clean the code so mistakenly removed these comments as well. I will add this back